angular.json

```diff
@@ -9,13 +9,24 @@
       "projectType": "application",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-devkit/build-angular:application",
           "options": {
-            "outputPath": "dist/browser",
-            "index": "./site/doc/index.html",
-            "main": "./site/doc/main.ts",
-            "tsConfig": "./site/doc/tsconfig.app.json",
-            "polyfills": "./site/doc/polyfills.ts",
+            "outputPath": "dist",
+            "index": "site/doc/index.html",
+            "browser": "site/doc/main.ts",
+            "server": "site/doc/main.server.ts",
+            "tsConfig": "site/doc/tsconfig.app.json",
+            "serviceWorker": "ngsw-config.json",
+            "prerender": {
+              "discoverRoutes": false,
```

Comment:
I think we can turn on the discoverRoutes option, which will hydrate more pages. WDYT?

Reply:
True, after investigationg a little bit and with the work on hydration from the Angular Team, passing this to true will not affect the performance

Let's do it :)